odbc/tests/metadata_tests.rs

```diff
@@ -0,0 +1,20 @@
+extern crate odbc_api;
+
+use crate::common::{connect, validate_rs};
+
+mod common;
+
+#[allow(dead_code)]
+//#[test]
+fn test_sql_tables_no_filter() {
```

Comment:
I've change the test file to use one that just connecting. Maybe this will make more sense to everyone. I also trimmed common.rs accordingly. The headers for test_default_connection still need to be flipped until connection is implemented on the driver.